### PR TITLE
Fix max listeners limit.

### PR DESCRIPTION
**Pull request by rcorral**
When rendering many dial chars, the Chart component listens to sidebar width change. If there's many charts, it'll throw a warning.

### CODE REVIEW

File: src/js/stores/SidebarStore.js [hunk 1]
```diff
@@ -99,4 +99,7 @@ class SidebarStore extends GetSetBaseStore {
 
 }
 
-module.exports = new SidebarStore();
+const store = new SidebarStore();
+store.setMaxListeners(1000);
```
Comment (jfurrow) [inline src/js/stores/SidebarStore.js]: Maybe we should use `Infinity` here?

**Comment by mlunoe** [comment]
Hmm... I wonder if we should not have this listener in the chart then?

**Comment by MatApple** [comment]
Since this is only to prevent a warning which will now only be shown in the 99th percentile I'm going to merge.

**Comment by mlunoe** [comment]
I don't think this a good solution. Could we get an actual solution for this?
I actually disagree with this entering the code base

**Comment by MatApple** [comment]
I agree it's not the *best* solution, but it is the simplest. What do you propose?

**Comment by mlunoe** [comment]
To not have the charts fire the sidebar action, but on the parent instead.

**Pull request merged**